application/validators/teams_validator.py

```diff
@@ -0,0 +1,32 @@
+""" Defines validators for team requests """
+
+from webargs import fields, ValidationError
+
+
+def index_args():
+    """Defines and validates params for index"""
+    return {}
+
+
+def create_args():
+    """Defines and validates params for create"""
+    return {"name": fields.Function(
+        required=True,
+        deserialize=validate_team_name
+    )}
+
+def update_args():
+    """Defines and validates params for update"""
+    return {
+        "id": fields.UUID(required=True, load_from='object_id', location='view_args'),
+        "name": fields.Function(deserialize=validate_team_name)
```

Comment:
this is rewriting inbuild webargs functionality, just do `fields.String(required=True)` The error messages will be autogenerated